src/mca/pdl/pdlopen/pdl_pdlopen_module.c

```diff
@@ -175,7 +175,26 @@ static int pdlopen_foreachfile(const char *search_path,
 
         dp = opendir(dirs[i]);
         if (NULL == dp) {
-            ret = PMIX_ERR_IN_ERRNO;
+            if (0 == i) {
```

Comment:
Yeah this was what I wanted to test today. In Open MPI `mca_base_component_path` provides a `:` separated list of paths. By default it contains two - one to the prefix install dir and the other to the user's home directory

> MCA mca base: parameter "mca_base_component_path" (current value: "/u/jjhursey/local/ompi/current-dev/lib/openmpi:/u/jjhursey/.openmpi/components", data source: default, level: 9 dev/all, type: string, synonyms: mca_component_path)

I just want to make sure we preserve the functionality that the `${HOME}/.openmpi` version is allowed to not exist, but at least one of the values must exist. I need to setup a build to confirm that this was preserved (I believe it was). I agree with Ralph that if we want something better we should talk to the OMPI community. But the OMPI version of this commit has not been merged yet, so we might be able to tackle it all at once.